pkg/controlplane/apiserver/server.go

```diff
@@ -146,6 +147,28 @@ func (c completedConfig) New(name string, delegationTarget genericapiserver.Dele
 		return nil, fmt.Errorf("failed to get listener address: %w", err)
 	}
 
+	if utilfeature.DefaultFeatureGate.Enabled(apiserverfeatures.CoordinatedLeaderElection) {
+		_ = s.VersionedInformers.Coordination().V1().Leases().Informer()
+		_ = s.VersionedInformers.Coordination().V1alpha1().LeaseCandidates().Informer()
+		// What's the best way of triggering these informer creations.
```

Comment:
something like you did. It's a little awkward pattern I agree. But it is what we have 🤷